main/converters/index.ts

```diff
@@ -0,0 +1,32 @@
+import path from 'path';
+import tempy from 'tempy';
+import {Encoding, Format} from '../common/types';
+import {track} from '../common/analytics';
+import h264Converters from './h264';
+import {ConvertOptions} from './utils';
+
+const converters = new Map([
+  [Encoding.h264, h264Converters]
+]);
```

Comment:
When we support recording in hevc, we can add converters for that here if they are different from h264 (same for ProRes etc)